hw/dv/verilator/cpp/mem_area.h

```diff
@@ -0,0 +1,91 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_HW_DV_VERILATOR_CPP_MEM_AREA_H_
+#define OPENTITAN_HW_DV_VERILATOR_CPP_MEM_AREA_H_
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+// This is the maximum width of a memory that's supported by the code in
+// prim_util_memload.svh
+#define SV_MEM_WIDTH_BITS 312
+#define SV_MEM_WIDTH_BYTES ((SV_MEM_WIDTH_BITS + 7) / 8)
+
+//
+// A "memory area" represents a memory in the simulated design.
+//
+
+class MemArea {
+ public:
+  // Constructor
```

Comment:
Can you make those doc strings doxygen-compatible, please? `/**` etc.

Reply:
Done. (And I even configured and ran Doxygen to make sure it looked ok). I'm not a huge fan: we're not generating HTML documentation for this code, so I've basically just taken the one and only source of documentation and made it harder to read. I guess this might be a plus if we decide to start using Doxygen in the future :-(